scripts/check-broken-links.mjs

```diff
@@ -73,42 +113,81 @@ async function checkBrokenLinks() {
 
       linksChecked.push(result)
 
-      if (result.state === 'BROKEN') {
+      // 403 errors returned to the crawler by and large work for web users.
+      if (result.state === 'BROKEN' && result.status !== 403) {
         brokenLinks.push(result)
       }
     })
+    .on('retry', (retryDetails) => {
```

Comment:
Not strictly necessary but at this stage it is good to have visibility into how retries are going.